sdk/cosmos/README.md

```diff
@@ -0,0 +1,295 @@
+﻿# Java SDK for SQL API of Azure Cosmos DB
```

Comment:
why are we adding Readme file as part of this file?

Dis this come as part of merging v3 branch to v4?

Reply:
I think so too. There are more changes on pom.xml files. I am planning to fix it now.